lib/xlsx.js

```diff
@@ -21,7 +19,7 @@ module.exports = {
       docs.map(stringify)
     );
 
-    XLSX.utils.book_append_sheet(workbook, worksheet, SHEET_NAME);
+    XLSX.utils.book_append_sheet(workbook, worksheet, 'aposDocs');
```

Comment:
Excel's default first sheet name is "Sheet1." We should use that convention rather than giving it a technical name.

Reply:
ok, i'll just remove it:

> If provided, the second argument is the name of the worksheet. If omitted, the default name "Sheet1" will be used.
https://docs.sheetjs.com/docs/api/utilities/wb